apps/server/src/controllers/v1/events.ts

```diff
@@ -44,4 +44,10 @@ router.delete('/:id', async (req: Request, res: Response) => {
     res.sendStatus(200)
 })
 
+router.get('/:id', async (req: Request, res: Response) => {
+    const event = await EventsModel.findById(req.params.id)
+
+    res.status(200).json(event.toJSON())
```

Comment:
toJSON 함수는 json 변환시 자동 실행됩니다
```suggestion
    res.status(200).json(event)
```